public/components/query_compare/search_result/result_components/result_panel.tsx

```diff
@@ -14,19 +14,27 @@ import {
 } from '@elastic/eui';
 
 import { ResultGridComponent } from './result_grid';
-import { SearchResults } from '../../../../types/index';
+import { QueryError, SearchResults } from '../../../../types/index';
 import { useSearchRelevanceContext } from '../../../../contexts';
 
 import './result_components.scss';
 
 interface ResultPanelProps {
   resultNumber: number;
   queryResult: SearchResults;
+  queryError: QueryError;
 }
 
-export const ResultPanel = ({ resultNumber, queryResult }: ResultPanelProps) => {
+export const ResultPanel = ({ resultNumber, queryResult, queryError }: ResultPanelProps) => {
   const { comparedResult1, comparedResult2 } = useSearchRelevanceContext();
 
+  const errorMessage = (
+    <>
+      <EuiHorizontalRule margin="s" />
+      <EuiText>Query Error</EuiText>
+    </>
+  );
```

Comment:
The error message style should be consistent with the existing ones. We should also add the real error message. I believe the 'query error' does not accurately reflect the actual behavior.

Reply:
I noticed in [this comment](https://github.com/opensearch-project/dashboards-search-relevance/pull/267#pullrequestreview-1603443181) you mentioned that OpenSearch errors should be in a toast component. Were you referring to this error message needing to be put in a toast?

Reply:
Yes, from the standpoint of UX, the very minimum is that all error messages be written in the same style for this plugin. And the best practice is to use toast, which is consistent to the overall OSD UX

Reply:
Offline conversation with @sejli. We believe toast is more commonly used in errors on browser interactions such as button clicks. For the time being, let's keep with the same styles for errors in query editors. In the long run, we may benefit from more involvement from the UX team @lauralexis.

Reply:
Adding on to our conversation, will be putting the error reasoning in the result grid while maintaining consistency with other error messages in the Search Comparison Tool.